src/open_dread_rando/misc_patches/actor_patcher.py

```diff
@@ -0,0 +1,62 @@
+import copy
+
+from json_delta import patch
+
+from open_dread_rando.patcher_editor import PatcherEditor
+
+
+def _modify_actor(editor: PatcherEditor, original_reference: dict, modifications: list,
+                  new_reference: dict, actor_groups: list, make_copy: bool = False):
+    is_in_place = original_reference == new_reference
+    scenario = editor.get_scenario(new_reference["scenario"])
+
+    if is_in_place:
+        actor = editor.resolve_actor_reference(original_reference)
+        make_copy = False
+    else:
+        actor = copy.deepcopy(editor.resolve_actor_reference(original_reference))
+        actor.sName = new_reference["actor"]
+        scenario.actors_for_layer(new_reference["layer"])[new_reference["actor"]] = actor
+
+    if modifications:
+        patch(actor, [
```

Comment:
I'd prefer just `import json_delta` at the top, and then `json_delta.patch()` here